### PR TITLE
This is a more general approach to support mods in ecwolf.

### PR DESCRIPTION
**Pull request by miwasp**
There are mods which use the same files as the original games and
there are no command line switches to tell ecwolf to use that files.
You have to put those mods in a subdir and cd into that subdir
before starting ecwolf.

### CODE REVIEW

File: packages/sx05re/emulators/ecwolf/scripts/ecwolf.sh [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 . /etc/profile
 
+# We cd into this directory before starting ecwolf
+RUN_DIR="/emuelec/configs/ecwolf"
```
Comment (shantigilbert) [inline packages/sx05re/emulators/ecwolf/scripts/ecwolf.sh:9]: Why not use $CONFIG_DIR? its the same path

Comment (shantigilbert) [reply packages/sx05re/emulators/ecwolf/scripts/ecwolf.sh:9]: Oh you overwrite it later on, I get it

Comment (miwasp) [reply packages/sx05re/emulators/ecwolf/scripts/ecwolf.sh:9]: You mean
RUN_DIR="${CONFIG_DIR}"

Comment (shantigilbert) [reply packages/sx05re/emulators/ecwolf/scripts/ecwolf.sh:9]: Yes but I was mistaken

**Pull request merged**